schemas/http.yml

```
@@ -8,6 +8,15 @@
   type: group
   fields:
 
+    - name: request.id
+      level: extended
+      type: keyword
+      short: HTTP request ID.
+      description: >
+        A unique identifier for each HTTP request.
```

Proposing we expand the description.

```suggestion
        A unique identifier for each HTTP request to correlate logs between clients
        and servers in transactions.
        
        The id may be contained in a non-standard HTTP header, such as `X-Request-ID`
        or `X-Correlation-ID`. 
```